app/models/user.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# User
+class User < ApplicationRecord
+  has_many :attendances, class_name: 'Attendance',
```

Comment:
Why did we need to put `class_name: 'Attendance`` here? The table name is already :attendances. We only have to put the class_name if the association is called by a different name.